site/app/templates/admin/UploadConfigForm.twig

```diff
@@ -35,30 +34,47 @@
     <br>
     <b>located in {{ target_dir }}</b>
     <br>&nbsp;<br>
+    <link>
+    <script>
+        function renameConfig(file_path){
+            var url = buildUrl({'component': 'admin', 'page': 'gradeable', 'action': 'check_being_used', 'config': file_path});
+            $('#alert_in_use').html('');
+            $('ol').empty();
```

Comment:
Maybe give your `ol` an id instead of querying for all `ol`'s  This could become a bug later on if any other `ol`s get used in the page.